data/explorations/adventure.yaml

```diff
@@ -350,7 +350,7 @@ states:
           refresher_exploration_id: null
         rule_specs:
         - inputs:
-            x: '{{numcoins}}'
+            x: 0
```

Comment:
Maybe let's just treat this the same as we do '2' -- no use keeping an invalid exploration and turning it into something that doesn't really flow correctly. (I think this one was originally meant to simulate a text adventure game with inventory.)